### ISSUE TITLE
No name is displayed for the input field in the other tab

### ISSUE THREAD

**Issue by FaCe2FaCe** [hperrin/svelte-material-ui#434]
**Describe the bug**

When going to a page that contains a Tabs component. Then switch to another tab and there is already something in the input field, the label is not displayed.

**To Reproduce**
1. Go to other site which contain a tabs component and select an other tab who the input component contain text, the label is not showing

**Screenshots**
![issue](https://user-images.githubusercontent.com/5448386/159804851-375aa410-dd0a-4ee1-9f25-2919b226c839.png)

**Desktop (please complete the following information):**
 - OS: Ubuntu 21.10 64-Bit 
 - Browser chromium
 - Version 99.0.4844.82 (Offizieller Build) snap (64-Bit)

**Additional context**
I use the smui version [4.2.0]

**Comment by FaCe2FaCe** [comment]
I just found a solution that looks like this:

<div class="m-3 mt-5 **relative**"  style="{((tabImport) ? CONaviStep != 1 : CONaviStep != 0) ? '**overflow: hidden; height: 0px;**' : ''}">

That's not the best but it works for me.

Problem description: When the page is called up, the width of the text field (outlined) is measured, which is then not done with hidden tabs.

**Comment by hperrin** [comment]
Yeah. The issue here is that the label needs to know its own dimensions to size itself correctly, and that can only happen when it's in the DOM and technically being rendered. That solution is probably the best one if you need it rendered when the tab isn't selected.
